code/modules/clothing/gloves/color.dm

```diff
@@ -9,9 +9,9 @@
 	burn_state = FIRE_PROOF
 
 /obj/item/clothing/gloves/color/yellow/power
-	description_antag = "These are a pair of power gloves, and can be used to fire bolts of electricity while standing over powered power cables."
 	var/old_mclick_override
 	var/datum/middleClickOverride/power_gloves/mclick_override = new /datum/middleClickOverride/power_gloves
+	antag_hints = "<span class='traitorhint'>These gloves have a set of hyperconductive threads woven into them. Use MMB while standing on a wire to arc flash your target.</span>"

```

Comment:
Took me a moment to get what MMB means. Also a bit weird how it is an OOC hint. Don't know how to say it better tho.

Reply:
I feel that mouse and key presses need a format to better know that's what they are, like a key press being [key]. Would that work with mouse buttons too?